orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -92,6 +116,29 @@ class ORTTrainerOptions(object):
                         'enable_adasum' : {
                             'type' : 'boolean',
                             'default' : False
+                        },
+                        'sliced_schema' : {
+                            "type": "dict",
+                            # the keys in `employee_eligibility` are strings matching this regex
```

Comment:
employee_eligibility?

Reply:
Errr........ Removed.

---
In reply to: [532624772](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r532624772) [](ancestors = 532624772)